lib/cocina/models.rb

```diff
@@ -49,6 +49,31 @@ class ValidationError < Error; end
     class Struct < Dry::Struct
       transform_keys(&:to_sym)
       schema schema.strict
+
+      def self.new(attributes = default_attributes, safe = false, &block)
+        super(underscore_attributes(attributes), safe, &block)
+      end
+
+      def self.underscore_attributes(attrs)
+        attrs.with_indifferent_access.deep_transform_keys(&:underscore)
+      end
+
+      # (temporarily?) allow callers to send camelCase method names to avoid having to change the world
+      def method_missing(method_name, *arguments, &block)
+        if respond_to_missing?(method_name)
+          public_send(method_name.to_s.underscore, *arguments, &block)
+        else
+          super
+        end
+      end
+
+      def respond_to_missing?(method_name, include_private = false)
+        respond_to?(method_name.to_s.underscore) || super
+      end
```

Comment:
Allow downstream codebases to continue calling camelCase methods on Cocina model instances (for now?).